src/components/misc/BoxColor/BoxColor.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+
+class BoxColor extends Component {
+
+    componentRgbToHex = (c) => {
+        let hex = c.toString(16);
+        return hex.length === 1 ? "0" + hex : hex;
+    }
+    rgbToHex = (r, g, b) => {
+        return `#${this.componentRgbToHex(r)}${this.componentRgbToHex(g)}${this.componentRgbToHex(b)} `
+    }
+
+    render() {
+        const {r, g, b} = this.props
+
+        return (
+            
+            <div style={{ backgroundColor:`rgb(${this.props.r},${this.props.g},${this.props.b})`}}>
```

Comment:
si ya habías hecho destructuring de `{r, g, b}` entonces no tienes que referenciarlos como `this.props.r`